dallinger/patches.py

```diff
@@ -0,0 +1,11 @@
+def patch_ipython():
+    try:
+        from ipykernel.iostream import OutStream
+    except ImportError:
+        return
+    else:
+        OutStream.writable = lambda self: True
+
+
+def patch():
```

Comment:
Why the alias? (No implied critique; genuinely curious.)

Reply:
Just to stop it filling up `__init__.py` with imports if more patches get added in future.